app/_src/production/install-kumactl.md

```diff
@@ -0,0 +1,87 @@
+
+{% tabs install-kumactl useUrlFragment=false %}
+{% tab install-kumactl Kubernetes %}
+1. Download {{site.mesh_product_name}}:
+{% tabs install_kumactl useUrlFragment=false %}
+{% tab install_kumactl Script %}
+
+Run the following script to automatically detect the operating system and download Kuma:
+
+<div class="language-sh">
+  <pre class="no-line-numbers"><code>curl -L https://kuma.io/installer.sh | VERSION={{ page.latest_version }} sh -</code></pre>
+</div>
+
+You can omit the `VERSION` variable to install the latest version.
+{% endtab %}
+{% tab install_kumactl Direct Link %}
+
+Download the distribution manually. Download a distribution for the **client host** from where you will be executing the commands to access Kubernetes:
+
+* <a href="https://download.konghq.com/mesh-alpine/kuma-{{ page.latest_version }}-centos-amd64.tar.gz">CentOS</a>
+* <a href="https://download.konghq.com/mesh-alpine/kuma-{{ page.latest_version }}-rhel-amd64.tar.gz">RedHat</a>
+* <a href="https://download.konghq.com/mesh-alpine/kuma-{{ page.latest_version }}-debian-amd64.tar.gz">Debian</a>
+* <a href="https://download.konghq.com/mesh-alpine/kuma-{{ page.latest_version }}-ubuntu-amd64.tar.gz">Ubuntu</a>
+* <a href="https://download.konghq.com/mesh-alpine/kuma-{{ page.latest_version }}-darwin-amd64.tar.gz">macOS</a> or run `brew install kumactl`
+
+and extract the archive with `tar xvzf kuma-{{ page.latest_version }}.tar.gz`
+{% endtab %}
+{% endtabs %}
+
+1. Add the `kumactl` executable to your path:
+```
+cd kuma-{{ page.latest_version }}/bin
+PATH=$(pwd):$PATH
+```
+
+{% endtab %}
+{% tab install-kumactl Helm %}
+Add the {{site.mesh_product_name}} charts repository:
+```sh
+helm repo add kuma https://kumahq.github.io/charts
+helm upgrade -i kuma kuma/kuma
+```
+
+{% endtab %}
+{% tab install-kumactl OpenShift %}
+Download {{site.mesh_product_name}}:
+```sh
+curl -L https://kuma.io/installer.sh | VERSION={{ page.latest_version }} sh -
+```
+{% endtab %}
+{% tab install-kumactl Docker %}
+Download {{site.mesh_product_name}}:
+```sh
+docker pull docker.io/kumahq/kuma-cp:{{ page.latest_version }}
+docker pull docker.io/kumahq/kuma-dp:{{ page.latest_version }}
+docker pull docker.io/kumahq/kumactl:{{ page.latest_version }}
+```
+{% endtab %}
+{% tab install-kumactl Linux %}
+Download {{site.mesh_product_name}} by doing one of the following:
+* Run the following script to automatically detect the operating system and download {{site.mesh_product_name}}:
+<div class="language-sh">
+<pre class="no-line-numbers"><code>curl -L https://kuma.io/installer.sh | VERSION={{ page.latest_version }} sh -</code></pre>
+</div>
+* <a href="https://download.konghq.com/mesh-alpine/kuma-{{ page.latest_version }}-{{ page.os }}-{{ page.arch }}.tar.gz">Download</a> the distribution manually, and then extract the archive with: `tar xvzf kuma-{{ page.latest_version }}`.
+
+{% tip %}
+Make sure you have tar and gzip installed.
```

Comment:
not sure if gzip is required, also I"m getting some weird errors when untaring it

```
$ tar xvzf kuma-0.0.0-preview.00c2d15ed-ubuntu-amd64.tar.gz                                                                                                                                                                                                      -- INSERT --
x ./: Can't restore time
x ./kuma-0.0.0-preview.00c2d15ed/
x ./kuma-0.0.0-preview.00c2d15ed/LICENSE
x ./kuma-0.0.0-preview.00c2d15ed/README
x ./kuma-0.0.0-preview.00c2d15ed/bin/
x ./kuma-0.0.0-preview.00c2d15ed/bin/kumactl
x ./kuma-0.0.0-preview.00c2d15ed/bin/kuma-prometheus-sd
x ./kuma-0.0.0-preview.00c2d15ed/bin/kuma-dp
x ./kuma-0.0.0-preview.00c2d15ed/bin/coredns
x ./kuma-0.0.0-preview.00c2d15ed/bin/kuma-cp
x ./kuma-0.0.0-preview.00c2d15ed/bin/envoy
x ./kuma-0.0.0-preview.00c2d15ed/NOTICE
x ./kuma-0.0.0-preview.00c2d15ed/conf/
x ./kuma-0.0.0-preview.00c2d15ed/conf/kuma-cp.conf.yml
x ./kuma-0.0.0-preview.00c2d15ed/NOTICE-kumactl
tar: Error exit delayed from previous errors.
```

FYI @lahabana since you are reworking the `distro.sh` script.

Reply:
Yeah the installer.sh does stuff to avoid this (and the news distros don't have this problem)

Reply:
@slonka I wonder if you're getting the errors because of the Kuma version: `kuma-0.0.0-preview`. I think it's displaying like that because you were on the `dev` version of the site. Once `dev` is replaced with 2.2.x, I think that might work because the `{{ page.latest_version }}` variable doesn't have a version to pull from for dev.